spec/neo4j/driver/internal_record_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+RSpec.describe Neo4j::Driver::Ext::InternalRecord do
+  subject(:res) do
+    session = driver.session
+    session.run("CREATE (p:Person{names: ['Mark', 'Paul']}) RETURN p.names AS names, ID(p) AS id").single
+  ensure
+    session&.close
+  end
+
+  its(:values) { are_expected.to include(%w[Mark Paul]) }
+  its(:first) { is_expected.to eq(%w[Mark Paul]) }
+  it { expect(res['names']).to eq(%w[Mark Paul]) }
```

Comment:
What was the reason for testing internal API? The difficulty is that we have 2 implementations which are identical only at the public API. If we want to unit test the internal implementation we would need 2 separate unit tests. Could this be avoided without any downside?

Reply:
Fair point.

Reply:
@lkalwa could we pick up this again?

Reply:
@klobuczek let's discard these changes.